src/controller/controller.ts

```diff
@@ -184,6 +184,18 @@ class Controller extends events.EventEmitter {
             databaseCoordinator.changeIeeeAddress(coordinator.ieeeAddr);
         }
 
+        if (startResult === 'resumed') {
+            const netParams = (await this.getNetworkParameters());
+
+            if (this.options.network.channelList[0] !== netParams.channel) {
+                debug.log(`Scheduling switch channel broadcast in 3min.`);
```

Comment:
Why do we wait 3 minutes before doing this? (why not e.g. 10 secs)

Reply:
I figured it was better to let things settle after init (to ensure the broadcast reached as many devices as possible). Since that worked exceptionally well on first try... I kept the 3 minutes after that, without looking back 😝
Probably should be adjusted after more tests. _Might even need a per-stack value if the post-init "ideal state" time isn't quite the same..._

Reply:
Lowered to 60sec:
- Triggering immediately on start (no setTimeout) causes collisions with permit join stuff, and post-init pings/broadcasts, creating unnecessary errors (although it still works fine, it's not great for user-feedback).
- It takes around 15 seconds for post-init stuff to be over on my very small test network => probably should allocate 30 minimum, and counting other stacks may need more time => 60... nice round number... 😅
- Tested half a dozen times with 60sec, consistent results every time.

Reply:
60 is fine then!